web-app/css/main.css

```diff
@@ -106,13 +106,13 @@ ul.one-to-many li.add {
 fieldset.embedded {
 	background-color: transparent;
 	border: 1px solid #CCCCCC;
+    box-shadow: none;
 	margin-left: 0;
 	margin-right: 0;
 	padding-left: 0;
 	padding-right: 0;
-	   -moz-box-shadow: none;
+	-moz-box-shadow: none;
```

Comment:
Also, prefixed versions must be together with the normal rule (that alignment in before version is silly though)